contracts/DCAHub/DCAHubSwapHandler.sol

```diff
@@ -142,17 +150,21 @@ abstract contract DCAHubSwapHandler is ReentrancyGuard, DCAHubConfigHandler, IDC
       _pairInSwap.tokenA = _tokens[indexTokenA];
       _pairInSwap.tokenB = _tokens[indexTokenB];
 
-      uint256 _magnitudeA = tokenMagnitude[_pairInSwap.tokenA];
```

Comment:
Moved it down because I was getting stack too deep errors 🤷

Reply:
A classic.